### ISSUE TITLE
Normally Private Screens Show (/mod/<modid>/moderation, /mod/<modid>/settings/analytics)

### ISSUE THREAD

**Issue by sylv256** [modrinth/code#2904]
### Please confirm the following.

- [x] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [x] I have tried resolving the issue using the [support portal](https://support.modrinth.com)

### What browsers are you seeing the problem on?

Firefox

### Steps to reproduce

1. Add `/settings/analytics` or `/moderation` to the end of a mod URL.
2. You will not see any private information.

### Expected behavior

You should not see this page. That is to say, the server should send a 403 page.

**Comment by Prospector** [comment]
Is not really an issue, there is no way to navigate to these pages naturally

**Comment by sylv256** [comment]
Yes, but most websites show a 403 Forbidden page to indicate to the user that they shouldn't be there. This is especially useful when the user is logged out but tries to load these pages.

**Comment by Prospector** [comment]
fixed
